cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -2425,43 +2414,69 @@ x"
         let e = parse(
             YaccKind::Grmtools,
             "%start A
+                %start A
+                %start A
```

Comment:
Are the multiple `%start`s intentional?

Reply:
Yeah, this is just a general testing for errors that produce multiple spans, and errors with multiple spans and multiple error kinds.
Previously we were using `DuplicateRule` as the 'multiple spans' error, I replaced that with `DuplicateStartRule`.

Reply:
Since we have another test with multiple `%start`s below, maybe it's not worth checking that here?

Reply:
Sure, I could just remove this one, and leave the other in place.

Reply:
Works for me!